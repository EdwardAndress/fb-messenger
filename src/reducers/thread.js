import { RECEIVE_THREAD } from '../actions/thread'
import { RECEIVE_MESSAGE, DELETE_MESSAGE } from '../actions/conversation'

export default function (state, action) {
  switch (action.type) {
    
  }
}
