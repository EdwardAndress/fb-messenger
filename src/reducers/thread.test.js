import {
  receiveThread,

} from '../actions/thread'
import thread from './thread'

/*

What BEHAVIOUR do we have to test?

Task 1, write the describe and its
Task 2, implement the expectations

*/

describe('Thread reducer', () => {
  it(`should add a message to the thread`, () => {
    expect(thread(null, receiveThread({ message: 'hello' }))).toMatchSnapshot()
  })
})
