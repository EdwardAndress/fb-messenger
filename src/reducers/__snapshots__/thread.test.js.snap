// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thread reducer should add a message to the thread 1`] = `
Object {
  "message": "hello",
}
`;
